com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKRayInteractorVisual.cs

```diff
@@ -14,33 +14,9 @@ namespace Microsoft.MixedReality.Toolkit.Input
     [DisallowMultipleComponent]
     [RequireComponent(typeof(LineRenderer))]
     [DefaultExecutionOrder(XRInteractionUpdateOrder.k_LineVisual)]
-    public class MRTKRayInteractorVisual : MonoBehaviour, IXRCustomReticleProvider
+    public class MRTKRayInteractorVisual : MonoBehaviour
```

Comment:
Isn't this probably supposed to be a derived class from the `MRTKBaseReticleVisual` you created?

Reply:
Also, why is there an MRTKRayInteractorVisual and an MRTKRayInteractorReticleVisual? These names are getting out of hand, too

Reply:
Whoops, I missed the subclassing!

THe names are getting pretty silly I agree. Thoughts on just doing `FarRayVisual` and `FarReticleVisual`?

Reply:
Done